src/Microsoft.Azure.WebJobs.Logging/Internal/Utility.cs

```diff
@@ -72,5 +74,80 @@ public static string DecodeBase64(string base64)
             string str = Encoding.UTF8.GetString(bytes);
             return str;
         }
+
+
+        private static async Task SafeCreateAsync(this CloudTable table)
```

Comment:
That would be good. It won't be testing your while(true) retry stuff though, ensuring that we don't go in to an infinite loop, etc.